clang/test/Analysis/int128-nocrash.c

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=optin.portability.UnixAPI \
+// RUN:    -triple x86_64-pc-linux-gnu -x c %s
+
+// Don't crash!
+// expected-no-diagnostics
+const __int128_t a = ( (__int128_t)1 << 64 );
```

Comment:
It'd be good to add a test case using `_BitInt(72)` or something along those lines to ensure we handle that properly as well.

Reply:
I added a test case. Let me know if you're ok with this. Thanks!